SaraAlert/SaraAlert
# SARAALERT-1098: Changing Jurisdiction to same value should not count as a transfer

# Description
Jira Ticket: [SARAALERT-1098](https://tracker.codev.mitre.org/browse/SARAALERT-1098)

While using the enrollment wizard if you changed the jurisdiction, clicked next and confirm the change, then click previous, click next again the same popup would appear to confirm the change again. 

This change will make it so a popup is only shown when changing the jurisdiction to a value that is different than the original value (last saved jurisdiction) or a new selection that is different than the last selection and the original value. 

# Testing
This fix was tested on the following browsers (submitter must check all those that apply):

1. Using the enrollment wizard fill out required sections until you can change jurisdictions. Change from state1 to 'state1, county1', click next, a popup should appear, confirm the changes. (different than the last saved and this is different than the last selected)
2. Click Previous, click next, no popup should appear (this is the last selected value)
3. Click Previous, change to 'state1, county2', click next, a popup should appear, confirm the changes. (this is different than the last selected and different than the last saved)
4. Click Previous, change to 'state1', click next, no popup should appear (this is the same as the last saved value)
